straxen/rundb.py

```diff
@@ -74,16 +65,7 @@ def __init__(self,
 
         self.hostname = socket.getfqdn()
 
-        if mongo_url is None:
-            mongo_url = get_mongo_url(self.hostname)
-
-        self.client = pymongo.MongoClient(mongo_url)
-
-        if mongo_dbname is None:
-            mongo_dbname = default_mongo_dbname
-        if mongo_collname is None:
-            mongo_collname = default_mongo_collname
-        self.collection = self.client[mongo_dbname][mongo_collname]
+        self.collection = utilix.rundb.pymongo_collection(**kwargs)
```

Comment:
The kwargs are  already used for the super().__init__

Reply:
This should be addressed now.